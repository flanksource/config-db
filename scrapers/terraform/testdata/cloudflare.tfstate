{
  "version": 4,
  "terraform_version": "1.9.4",
  "serial": 118,
  "lineage": "bef3a10e-9947-edde-9ccd-79f630c9ef13",
  "outputs": {},
  "resources": [
    {
      "mode": "data",
      "type": "sops_file",
      "name": "cloudflare_secrets",
      "provider": "provider[\"registry.terraform.io/carlpett/sops\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "data": "flanksource.com",
            "id": "-",
            "input_type": null,
            "source_file": "secret.sops.yaml"
          },
          "sensitive_attributes": [
            [
              {
                "type": "get_attr",
                "value": "data"
              }
            ]
          ]
        }
      ]
    }
  ],
  "check_results": null
}
