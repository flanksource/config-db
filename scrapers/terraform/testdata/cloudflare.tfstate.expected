{
  "version": 4,
  "terraform_version": "1.9.4",
  "serial": 118,
  "lineage": "bef3a10e-9947-edde-9ccd-79f630c9ef13",
  "outputs": {},
  "resources": [
    {
      "mode": "data",
      "type": "sops_file",
      "name": "cloudflare_secrets",
      "provider": "provider[\"registry.terraform.io/carlpett/sops\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "data": "sha256(8e2e88380fc2685aa448d2ad4725e9bc15d232d89c3e027e3504be135d853bfb)",
            "id": "-",
            "input_type": null,
            "source_file": "secret.sops.yaml"
          },
          "sensitive_attributes": [
            [
              {
                "type": "get_attr",
                "value": "data"
              }
            ]
          ]
        }
      ]
    }
  ],
  "check_results": null
}
